spec/lib/dhis2/api/data_element_spec.rb

```diff
@@ -37,13 +37,13 @@
         type:                 "int",
         aggregation_operator: "SUM",
         zero_is_significant:  true,
-        category_combo: {
+        category_combo:       {
           id: "p0KPaWEg3cf", name: "default"
         }
       }
     end
 
-    let(:elements) { [ data_element_1, data_element_2 ] }
+    let(:elements) { [data_element_1, data_element_2] }
```

Comment:
I would add a spec for data element to create without the default values to cover the "fetch" with default.